package/MDAnalysis/coordinates/GRO.py

```diff
@@ -96,7 +96,7 @@
 
 
 .. _Gromacs: http://www.gromacs.org
-.. _GRO: http://manual.gromacs.org/current/online/gro.html
+.. _GRO: https://manual.gromacs.org/2024.4/reference-manual/file-formats.html#gro
```

Comment:
```suggestion
.. _GRO: https://manual.gromacs.org/current/reference-manual/file-formats.html#gro
```
Best to not hardcode a version where possible.